docs/fabric-native-components-android.md

```diff
@@ -0,0 +1,439 @@
+---
+id: fabric-native-components-android
+title: 'Fabric Native Modules: Android'
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+React Native uses Gradle to manage dependencies, auto-linking and codegen. To use this, we're going to add the `RTNCenteredText` Fabric Native Component to our `Demo` application.
+
+### 1. Link RTNCenteredText as an npm dependency:
```

Comment:
this step should be removed. For consistency, we should show how to create a Fabric Component as part of the application